{
  "id": 1356,
  "name": "ERROR_CLUSTER_DATABASE_SEQMISMATCH",
  "description": "The join operation failed because the cluster database sequence number has changed or is incompatible with the locker node. This may happen during a join operation if the cluster database was changing during the join.",
  "external_url": "https://github.oblokos.com/?art=ERROR_CLUSTER_DATABASE_SEQMISMATCH",
  "image": "https://github.oblokos.com/art/ERROR_CLUSTER_DATABASE_SEQMISMATCH.png",
  "opensea_url": "https://opensea.io/assets/matic/0x252f9390feb63aeb00042dd5c4ff84af33f42b85/1356",
  "attributes": [
    {
      "trait_type": "artist",
      "value": "0x5Ae65e30834f61B26A25D9F9F0847c3da11dD391"
    },
    {
      "trait_type": "collection",
      "value": "Metaverse First Contact"
    },
    {
      "trait_type": "error_code",
      "value": "5083 (0x13DB)"
    },
    {
      "display_type": "number",
      "trait_type": "frame",
      "value": "05282"
    },
    {
      "display_type": "number",
      "trait_type": "index",
      "value": 0.534
    },
    {
      "display_type": "number",
      "trait_type": "level",
      "value": 0
    }
  ]
}